### ISSUE TITLE
Exiting due to exception: Incompatible shapes: 

### ISSUE THREAD

**Issue by imucsliurui** [keithito/tacotron#47]
I train the model use my own data.
But after 28 steps  I got the following error that I have no idea how to slove:
can you tell me the reason, thank you very much!
Best Regards!
=================================================================

[2017-09-25 22:50:39.149]  Generated 32 batches of size 32 in 1.763 sec
[2017-09-25 22:50:43.874]  Step 24      [4.483 sec/step, loss=0.90163, avg_loss=0.94017]
[2017-09-25 22:50:48.887]  Step 25      [4.505 sec/step, loss=0.91800, avg_loss=0.93929]
[2017-09-25 22:50:50.143]  Step 26      [4.380 sec/step, loss=0.91692, avg_loss=0.93842]
[2017-09-25 22:50:50.960]  Step 27      [4.248 sec/step, loss=0.89417, avg_loss=0.93679]
[2017-09-25 22:50:52.714]  Step 28      [4.159 sec/step, loss=0.89980, avg_loss=0.93547]
[2017-09-25 22:50:55.308]  Exiting due to exception: Incompatible shapes: [32,1070,80] vs. [32,1000,80]
	 [[Node: model/loss/sub = Sub[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"](datafeeder/input_queue_Dequeue/_555, model/inference/Reshape)]]
	 [[Node: model/inference/post_cbhg/dense/Tensordot/Shape/_1109 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_11312_model/inference/post_cbhg/dense/Tensordot/Shape", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

Caused by op 'model/loss/sub', defined at:
  File "montrain.py", line 156, in <module>
    main()
  File "montrain.py", line 151, in main
    train(log_dir, args)
  File "montrain.py", line 65, in train
    model.add_loss()
  File "/home/liurui/workspace/keithito-tacotron-master/models/tacotron.py", line 113, in add_loss
    self.mel_loss = tf.reduce_mean(tf.abs(self.mel_targets - self.mel_outputs))
  File "/home/liurui/anaconda2/envs/keithito-tacotron/lib/python3.5/site-packages/tensorflow/python/ops/math_ops.py", line 865, in binary_op_wrapper
    return func(x, y, name=name)
  File "/home/liurui/anaconda2/envs/keithito-tacotron/lib/python3.5/site-packages/tensorflow/python/ops/gen_math_ops.py", line 2629, in _sub
    result = _op_def_lib.apply_op("Sub", x=x, y=y, name=name)
  File "/home/liurui/anaconda2/envs/keithito-tacotron/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/home/liurui/anaconda2/envs/keithito-tacotron/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 2630, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/home/liurui/anaconda2/envs/keithito-tacotron/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1204, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): Incompatible shapes: [32,1070,80] vs. [32,1000,80]
	 [[Node: model/loss/sub = Sub[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"](datafeeder/input_queue_Dequeue/_555, model/inference/Reshape)]]
	 [[Node: model/inference/post_cbhg/dense/Tensordot/Shape/_1109 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_11312_model/inference/post_cbhg/dense/Tensordot/Shape", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

**Comment by keithito** [comment]
You can fix this by setting the max_iters hyperparameter to a larger value, for example by adding the flag --hparams="max_iters=400".

Duplicate of #16

**Comment by keithito** [comment]
Duplicate of #16

**Comment by imucsliurui** [comment]
Thank you very much!
